aidlc/spaces/default/intents/260718-api-workers-migration/construction/api-worker/infrastructure-design/infrastructure-design-questions.md

```
@@ -0,0 +1,15 @@
+
+インフラ判断は全て確定済み: 構成の正本 = `server/api/wrangler.jsonc`（C-3、実体確認済み）、dev のみデプロイ（Q8=B）、Terraform 凍結、リソースは既存（R2 バケット・KV DO 作成済み、#1089/#1093）。application-design 成果物（components.md / services.md）は infra スコープにより不在（expected）— nfr-design の logical-components.md がコンポーネント正本。
+
+新規リソースの作成・変更を伴わない（既存 binding の設計固定 + `RUNTIME` var 追加のみ）ため、人間への確認事項はない。
+
+---
+
+## G1. Approval gate (typed turn required)
+
+Infrastructure Design 5成果物（READY）の承認ゲート。チャットに approve / request changes をタイプしてください。
+
+[Answer]: approve (typed, 2026-07-18)
```

markdownlint-cli2 (0.23.0)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "answer"

(MD053, link-image-reference-definitions)